package.json

```diff
@@ -46,19 +42,20 @@
     "mysql": "^2.18.1",
     "node-fetch": "^2.6.1",
     "typescript": "~3.9.3",
-    "vue-cli-plugin-express": "^1.0.2"
+    "ts-node": "^9.1.1"
   },
   "devDependencies": {
     "@types/node": "^14.0.13",
     "@typescript-eslint/eslint-plugin": "^2.33.0",
     "@typescript-eslint/parser": "^2.33.0",
-    "@vue/cli-plugin-router": "^4.4.0",
-    "@vue/cli-plugin-vuex": "^4.4.0",
     "@vue/eslint-config-prettier": "^6.0.0",
     "@vue/eslint-config-typescript": "^5.0.2",
     "eslint-plugin-prettier": "^3.1.3",
     "eslint-plugin-vue": "^6.2.2",
-    "prettier": "^1.19.1",
-    "vue-template-compiler": "^2.6.11"
+    "nodemon": "^2.0.7",
+    "prettier": "^1.19.1"
```

Comment:
@ChaituVR Can you also remove @vue/eslint-config-prettier and @vue/eslint-config-typescript?

Reply:
@bonustrack Removed them and all other vue related lint packages, seems to be working the same as before, please let me know if something is missing :)